package/src/contexts/messagesContext/MessagesContext.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { PropsWithChildren, useContext } from 'react';
 
-import type { TouchableOpacityProps } from 'react-native';
+import { FlatList, TouchableOpacityProps } from 'react-native';
```

Comment:
```suggestion
import type { TouchableOpacityProps } from 'react-native';
```

I see that FlatList is passed to message context now... which is absolutely unnecessary anymore, please remove it from the context